app/pb_kits/playbook/pb_user_badge/badges/_badges.scss

```diff
@@ -0,0 +1,11 @@
+.veteran-st0{fill:#004876;}
+.veteran-st1{fill:#FFFFFF;}
+.veteran-st2{fill:#BB2432;}
+
+
+.st0{fill:url(#SVGID_1_);}
+.st1{fill:url(#SVGID_2_);}
+.st2{fill:url(#SVGID_3_);}
+.st3{fill:url(#SVGID_4_);}
+.st4{fill:url(#SVGID_5_);}
+.st5{fill:url(#SVGID_6_);}
```

Comment:
Why is this required? Do we need to generate a new SVG?

Reply:
Removed this file and added styles back to the svg files